client/my-sites/backup/hooks.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * External dependencies
+ */
+import { isArray } from 'lodash';
+import { useEffect, useMemo, useRef } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { applySiteOffset } from 'calypso/lib/site/timezone';
+import { getHttpData } from 'calypso/state/data-layer/http-data';
+import { getRequestActivityLogsId, requestActivityLogs } from 'calypso/state/data-getters';
+import { requestRewindCapabilities } from 'calypso/state/rewind/capabilities/actions';
+import getRewindCapabilities from 'calypso/state/selectors/get-rewind-capabilities';
+import getSiteGmtOffset from 'calypso/state/selectors/get-site-gmt-offset';
+import getSiteTimezoneValue from 'calypso/state/selectors/get-site-timezone-value';
+import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
+
+const isLoading = ( response ) => [ 'uninitialized', 'pending' ].includes( response.state );
+
+const byActivityTsDescending = ( a, b ) => ( a.activityTs > b.activityTs ? -1 : 1 );
+
+export const SUCCESSFUL_BACKUP_ACTIVITIES = [
+	'rewind__backup_complete_full',
+	'rewind__backup_complete_initial',
+	'rewind__backup_only_complete_full',
+	'rewind__backup_only_complete_initial',
+];
+
+export const BACKUP_ATTEMPT_ACTIVITIES = [
+	...SUCCESSFUL_BACKUP_ACTIVITIES,
+	'rewind__backup_error',
+];
+
+export const DELTA_ACTIVITIES = [
+	'attachment__uploaded',
+	'attachment__deleted',
+	'post__published',
+	'post__trashed',
+	'plugin__installed',
+	'plugin__deleted',
+	'theme__installed',
+	'theme__deleted',
+	'user__invite_accepted',
+];
```

Comment:
In `client/lib/jetpack/backup-utils.js`, we use another constant that is very similar but has more activities, is there a reason not to use this constant in that file?